features/all-positions/AllPositions.tsx

```diff
@@ -267,6 +322,11 @@ const AllPositions = () => {
             </PageButtons>
           </TableContainer>
         </Box>
+        <PositionActionsDialog
+          isDialogShowing={isDialogShowing}
+          handleClose={toggleDialog}
```

Comment:
This will close the dialog on click away right?

Reply:
yup. if you click away it calls this `toggleDialog` function.